### PR TITLE
add five random recipes

### PR DESCRIPTION
**Pull request by vadim9999**

### CODE REVIEW

File: src/components/Modals/ModalWork.js [hunk 1]
```diff
@@ -46,10 +46,10 @@ class Modals extends Component {
   }
 
   displayIngredients() {
-    let recipes = data.getRecipe();
+
     // console.log(recipes);
-    let random_key = Math.floor(Math.random() * 101);
-    let result = recipes[random_key]["ingredients"];
+
+    let result = this.props.recipe;
```
Comment (atherdon) [inline src/components/Modals/ModalWork.js]: i think here we return an ingredients, not recipe

File: src/components/WorkSchedule/ColumnRender.js [hunk 2]
```diff
@@ -11,7 +11,8 @@ class ColumnRender extends Component {
     var result = data.getDishByWeek(this.props.randomWeek)[0]["recipes"][
       this.props.index + 1
     ];
-    return rendererFunc(result, this.props.mode);
+
+    return rendererFunc(result, this.props.mode, this.props.randomRecipes[this.props.index]);
```
Comment (atherdon) [inline src/components/WorkSchedule/ColumnRender.js]: this function only making more and more complexier for me

File: src/components/WorkSchedule/Methods.js [hunk 3]
```diff
@@ -76,14 +77,16 @@ function rendererFunc(daySchedule, mode) {
...
     if (item === buffer[i].start) {
       if (i < buffer.length - 1) {
         i += 1;
       }
       if (j < buffer.length) {
         j += 1;
       }
-      return <Modals mode={mode} data={buffer[j - 1]} key={item} />;
+
+      return <Modals mode={mode} data={buffer[j - 1]} key={item} recipe={recipe}/>;
```
Comment (atherdon) [inline src/components/WorkSchedule/Methods.js]: looks complex. let's think how we can make it better

File: src/components/WorkSchedule/Schedule.js [hunk 4]
```diff
@@ -17,7 +17,7 @@ class ScheduleRender extends Component {
         <div key={index} className="data-class">
           <ListGroup>
             <ListGroupItem style={defaultBlockHeight}>{day}</ListGroupItem>
-            <ColumnRender mode={this.props.mode} key={shortid.generate()} index={index}  randomWeek={this.props.week}/>
+            <ColumnRender mode={this.props.mode} key={shortid.generate()} index={index}  randomWeek={this.props.week} randomRecipes={this.props.randomRecipes}/>
```
Comment (atherdon) [inline src/components/WorkSchedule/Schedule.js]: very scary

File: src/components/WorkSchedule/WorkSchedule.js [hunk 5]
```diff
@@ -29,6 +29,19 @@ class WorkSchedule extends Component {
     return "week" + Math.floor(Math.random() * 10);
   }
 
+  getFiveRandomRecipes(){
```
Comment (atherdon) [inline src/components/WorkSchedule/WorkSchedule.js]: it's give you five recipes or ingredients?

Comment (vadim9999) [reply src/components/WorkSchedule/WorkSchedule.js]: I need rename on ingredients

File: src/components/WorkSchedule/ColumnRender.js (hunk 2 above)
Comment (vadim9999) [reply src/components/WorkSchedule/ColumnRender.js]: How can I make it easier?

Comment (atherdon) [reply src/components/WorkSchedule/ColumnRender.js]: it's a hard question. First of all - i can guess that this function is heavy - so the easiest way - to split it into a few functions. Second thing - we have a 3 params, and they are long. i mean - i can bet that we have a lot of similar data inside this params. let's make this a separated task.

Comment (atherdon) [reply src/components/WorkSchedule/ColumnRender.js]: let's skip it for now - buzz me when you fix my comments to this PR so i can merge it

**Comment by vadim9999** [comment]
What do you think about it?

**Comment by atherdon** [comment]
Ohh, i got your idea about class. Yep, it's better.
but again, please print each of object field by using `console.log` and compare them. By my assumption - there some similar data inside of it...

**Comment by atherdon** [comment]
should i merge this request or we still have pending changes?

**Comment by vadim9999** [comment]
Need some changes

**Comment by atherdon** [comment]
Cool. Keep me posted #50

**Comment by vadim9999** [comment]
you can merge this code

**Pull request merged**